src/Revenue/Repositories/Revenue.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace Give\Revenue\Repositories;
+
+use InvalidArgumentException;
+
+/**
+ * Class Revenue
+ * @package Give\Revenue\Repositories
+ *
+ * Use this class to get data from "give_revenue" table.
+ *
+ * @since 2.9.0
+ */
+class Revenue {
+	/**
+	 * Revenue constructor
+	 */
+	public function constructor() {
```

Comment:
Remove the empty constructor if you don't plan to use it.

Reply:
fixed: https://github.com/impress-org/givewp/compare/f2d051c89917...bdedb19969be